controller/src/controller/models/domain/rsptag.py

```diff
@@ -576,3 +578,86 @@ def subset(
 
         # Return the results.
         return type(self)(tags)
+
+    def by_type(self, image_type: RSPImageType) -> Self:
+        """Return all tags of a given type.
+
+        Parameters
+        ----------
+        image_type
+            RSPImageType to select from collection.
+
+        Returns
+        -------
+        RSPImageCollection
+            Collection of tags matching the given type.
+        """
+        return type(self)(self._by_type[image_type])
+
+    def apply_policy(self, policy: RSPImageFilterPolicy) -> Self:
```

Comment:
I would call this method `filter`. That's exactly what it does; this seems a bit wordy.